linkerd/app/inbound/src/policy/tcp.rs

```diff
@@ -169,3 +176,22 @@ where
         }))
     }
 }
+
+/// Checks whether the destination port's `AllowPolicy` is authorized to
+/// accept connections given the provided TLS state.
+fn check_authorized(
+    server: &ServerPolicy,
+    dst: OrigDstAddr,
```

Comment:
nit, not a big deal: it kinda feels like this should just take an `&AllowPolicy` instead of two pieces of it (the `ServerPolicy` and orig dst)... it seems like the only reason it doesn't is because we want to borrow the `ServerPolicy` for the tracing event, but...we could just move that into this function?

Reply:
or maybe it's like that for the tests? in which case, no worries. not a big deal regardless.

Reply:
yeah, i want to test this function without all of the extra watch stuff.